var/spack/repos/builtin/packages/binutils/package.py

```diff
@@ -123,6 +124,16 @@ def configure_args(self):
 
         return args
 
+    @when('+static')
+    def build(self, spec, prefix):
+        params = ['V=1']
+        params += self.build_targets
+        with working_dir(self.build_directory):
+            make(*params)
+            make('clean')
```

Comment:
`make; make clean` looks like a noop. Is it explained somewhere why this is needed (or does the build fail without that)?

Reply:
It's actually the configure step that fails with `-all-static`, so this first configured without `-all-static`, cleaned up and then compiled with `-all-static` (which didn't configure again). I have changed this to do the configure explicitly, which hopefully makes this clearer.